### PR TITLE
PLUGIN-1495 PLUGIN-1496 : Fix schema validation issues

### PR DESCRIPTION
**Pull request by sgarg-CS**
[PLUGIN-1495](https://cdap.atlassian.net/browse/PLUGIN-1495) 
[PLUGIN-1496](https://cdap.atlassian.net/browse/PLUGIN-1496)

### CODE REVIEW

File: confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSource.java [hunk 1]
```diff
@@ -102,32 +99,25 @@ private Schema inferSchema(FailureCollector failureCollector) {
...
-          keySchemaShouldBeAdded = false;
-        } else if (field.getName().equals(conf.getValueField())) {
-          Schema valueSchema = fetchSchema(schemaRegistryClient, conf.getTopic() + "-value");
-          newFields.add(Schema.Field.of(field.getName(), valueSchema));
-          messageSchemaShouldBeAdded = false;
-        } else {
-          newFields.add(field);
-        }
-      }
+      boolean messageSchemaShouldBeAdded = conf.getvalueSchema() != null;
```
Comment (dhiraj0116) [inline confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSource.java]: Is it only used once?

File: confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSource.java [hunk 2]
```diff
@@ -102,32 +99,25 @@ private Schema inferSchema(FailureCollector failureCollector) {
...
+          newFields.add(Schema.Field.of(conf.getvalueSchema(), valueSchema));
+        } else {
+          failureCollector.addFailure(String.format("Invalid value schema name '%s'.", conf.getvalueSchema()),
+                                      null)
+            .withConfigProperty(ConfluentStreamingSourceConfig.NAME_VALUE_SCHEMA);
+          throw failureCollector.getOrThrowException();
+        }
       }
-      return Schema.recordOf(initialSchema.getRecordName(), newFields);
+      return Schema.recordOf("output", newFields);
```
Comment (dhiraj0116) [inline confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSource.java]: Can we use constant here?

File: confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSource.java (hunk 1 above)
Comment (sgarg-CS) [reply confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSource.java]: the variable `messageSchemaShouldBeAdded`?

File: confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSource.java (hunk 2 above)
Comment (sgarg-CS) [reply confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSource.java]: Yes, I've added a constant: `NAME_OUTPUT` 07d7595095cb3477bff92e36769f6ee73a656ee1

**Pull request merged**